aldryn_newsblog/admin.py

```diff
@@ -45,6 +45,7 @@ class ArticleAdminForm(TranslatableModelForm):
 
     class Meta:
         model = models.Article
+        fields = '__all__'
```

Comment:
@FinalAngel that is a very bad practice. See https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#selecting-the-fields-to-use and https://github.com/blog/1068-public-key-security-vulnerability-and-mitigation